projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -0,0 +1,146 @@
+# Meta Jobs
+
+Express dependecies between data jobs.
+
+A plugin for Versatile Data Kit which extends its Job API with an additional feature which allows users to trigger so called Meta Jobs.
+Meta Jobs are data jobs which trigger one or more other jobs, wait for their completion, and then trigger another set of jobs until either the entire job pipeline has succeeded.
+
+
+## Usage
+
```

Comment:
maybe add information about any existing alternative approach using the current control-service API, and comparison?

Reply:
README is not a feature spec. Definitely not here if anywhere since I have not investigated alternative approaches.

Reply:
our own execution api is one

Reply:
in fact, it is not mentioned, probably purposefully or not